Tools/CISupport/ews-app/ews/views/submittoews.py

```diff
@@ -45,11 +45,17 @@ def get(self, request):
     def post(self, request):
         try:
             change_id = request.POST.get('change_id')
+            _log.info('SubmitToEWS::change: {}'.format(change_id))
             change_id = int(change_id)
         except:
             return HttpResponse('Invalid patch id provided, should be an integer. git hashes are not supported.')
 
-        _log.info('SubmitToEWS::change: {}'.format(change_id))
+        if change_id < 459500:
+            return HttpResponse('Patch is too old. Skipping.')
+
+        if change_id > 600000:
```

Comment:
Setting a time limit on deprecating patch based workflows, eh?

Reply:
Yeah 😆 
Did very rough calculations. It's 10+ year in future based on current patch rate.